sdk/communication/Azure.Communication.CallAutomation/src/Models/AddParticipantsOptions.cs

```diff
@@ -18,6 +18,7 @@ public class AddParticipantsOptions
         public AddParticipantsOptions(IEnumerable<CommunicationIdentifier> participantsToAdd)
         {
             ParticipantsToAdd = participantsToAdd;
+            RepeatabilityHeaders = new RepeatabilityHeaders();
```

Comment:
is this needed? If so, don't we need it for other `Options` classes too?

Reply:
Yes - thank you for pointing out. we need for all options